packages/pp-binary-ints/pp-binary-ints.0.1.0/opam

```diff
@@ -0,0 +1,46 @@
+opam-version: "2.0"
+synopsis: "Pretty Printing Binary Integers"
+description: """
+Pretty Printing Binary Integers
+This library contains functions for pretty printing integers as
+unsigned binary integers.
+"""
+maintainer: ["Ifaz Kabir"]
+authors: ["Ifaz Kabir"]
+license: "CC0-1.0"
+tags: ["printf" "format" "boolean" "binary"]
+homepage: "https://github.com/ifazk/pp-binary-ints"
+doc: "https://ifazk.github.io/pp-binary-ints/"
+bug-reports: "https://github.com/ifazk/pp-binary-ints/issues"
+depends: [
+  "dune" {>= "2.9"}
+  "ocaml" {>= "4.10" & < "4.11" | > "4.11" & < "4.14"}
```

Comment:
```suggestion
  "ocaml" {>= "4.10" & < "4.14"}
```
The way the bounds are written basically does nothing, and the packages was compiled and tested successfully on 4.11 by the CI: https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/5f209f8c09072673acdd2931e914db2c10f9dd25/variant/opam-2.0,compilers,4.11,pp-binary-ints.0.1.0